src/main/java/seedu/address/logic/commands/quiz/QuizCheckAnswer.java

```diff
@@ -56,9 +56,9 @@ public CommandResult execute(Model model) throws CommandException {
         model.addQuizResult(quizResult);
 
         if (result) {
-            return new CommandResult(ANSWER_CORRECT);
+            return new CommandResult(ANSWER_CORRECT, false, false, true, false);
         } else {
-            return new CommandResult(ANSWER_WRONG);
+            return new CommandResult(ANSWER_WRONG, false, false, true, false);
         }
```

Comment:
There are four boolean fields in one of the constructors of `CommandResult` I am thinking of replacing it with a two-argument `CommandResult(String, int)`, where the `int` can hold several flags at once, and then using bitwise operations to extract the boolean variables.